tools/util/profiles/cqlstress_bigint_profile.yaml

```diff
@@ -0,0 +1,19 @@
+table: test_table
+
+table_definition:
+  CREATE TABLE IF NOT EXISTS test_table (
+    pkey blob PRIMARY KEY,
+    c1 bigint,
+    c2 bigint,
+    c3 bigint
+   );
+
+keyspace: cqlstress_bigint_keyspace
+
+keyspace_definition: |
+  CREATE KEYSPACE IF NOT EXISTS cqlstress_bigint_keyspace 
+  WITH replication = { 'class': 'SimpleStrategy', 'replication_factor': 1};
```

Comment:
Should we be still using `SimpleStrategy` in our internal code, such as in cql-stress implementation?
@piodul @Lorak-mmk

Reply:
I think it should not be the default (and it is used in all profiles right now, so that should be changed) but there should be some test cases for it as long as it exists in Scylla / C*

Reply:
Notice that these profiles are only used for tests.

BTW, related question: as of now, the cluster used in the CI consists of only one node. Maybe it's something to be changed? Maybe we should use a 3-node cluster. Then it would make sense to use/test some other replication strategies.

Reply:
Even if it's  only tests we should use NTS by default.
Yes, bigger cluster in CI sounds good

Reply:
Created an issue: https://github.com/scylladb/cql-stress/issues/88